presto-docs/src/main/sphinx/release/release-0.237.rst

```diff
@@ -0,0 +1,62 @@
+=============
+Release 0.237
+=============
+
+General Changes
+_______________
+* Fix NPE in common sub-expression optimization triggered by CASE-WHEN expression.
+* Fix compiler failure with function type when common sub-expression optimization is enabled.
+* Add JSON format for `EXPLAIN` with type `LOGICAL` and `DISTRIBUTED`.
+* Add functions :func:`array_sum` and :func:`map_normalize`.
+* Add new session property ``warning_handling `` to control how warnings are handled. The options are ``SUPPRESS``, ``NORMAL`` and ``AS_ERROR``. The default value is ``NORMAL``.
+* Add support for Druid's basic and kerberos authentication.
+* Add support for defining SQL-invoked functions in plugins.
+* Added a new aggregation function SET_AGG<T> that returns ARRAY<T> - equivalent to ARRATY_AGG(DISTINCT).
+* COUNT(DISTINCT) suggests using approx_distinct instead.
+* Introducing NodeStatusService interface to find out if a worker is in maintenance mode or ready to take tasks. Default behavior is no op. But can be implemented and override the
+  behavior. Config controlling the behavior `node-status-service.enabled`.
+* Optional NodeStatusService to check Nodes status (i.e. Available/In Maintenance) through some external source and mark them Inactive if not available to process queries.
+* Support listing functions whose names match a specified pattern using the SHOW FUNCTION LIKE syntax.
+* TaskStatus JSON decoding improved about 15% CPU by making it leaner.
```

Comment:
may be putting this as:
`Improved TaskStatus JSON decoding by making it leaner`